message-bus-tester/package.json

```diff
@@ -11,6 +11,7 @@
   "author": "Libero Reviewer Team",
   "license": "ISC",
   "dependencies": {
+    "event-bus": "../lib/event-bus",
```

Comment:
I think we need to add collection prefixes such as `@libero/` to local packages to make it clear they're not the npm published one where used. See https://www.npmjs.com/package/event-bus

Reply:
Also see https://github.com/libero/reviewer/pull/237#discussion_r323271032. I didn't refresh before reviewing commented on that PR after merge